libs/prelude/Prelude.idr

```diff
@@ -214,6 +214,30 @@ Enum Char where
 
   pred c = fromNat (pred (toNat c))
 
+Enum Bits8 where
+  toNat b   = fromIntegerNat (prim__zextB8_BigInt b)
```

Comment:
I (mistakenly, I think) initially tried to create an instance for Bits n, thinking this was where Bits8 came from. But it's not. The code I was writing needed an instance of Bits8, not Bits 8.